src/H5FD.c

```diff
@@ -928,9 +928,10 @@ H5FD_cmp(const H5FD_t *f1, const H5FD_t *f2)
 {
     int ret_value = -1; /* Return value */
 
-    FUNC_ENTER_NOAPI_NOERR /* return value is arbitrary */
+    FUNC_ENTER_NOAPI_NOERR; /* return value is arbitrary */
```

Comment:
The semicolon here prevents the formatter from botching the subsequent lines of code.